src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/CommandHandlers/UpdateChangeOfPartyRequestCommandHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NServiceBus;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Data.Extensions;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.CommandHandlers
+{
+    public class UpdateChangeOfPartyRequestCommandHandler : IHandleMessages<UpdateChangeOfPartyRequestCommand>
+    {
+        private readonly Lazy<ProviderCommitmentsDbContext> _dbContext;
+
+        public UpdateChangeOfPartyRequestCommandHandler(Lazy<ProviderCommitmentsDbContext> dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public async Task Handle(UpdateChangeOfPartyRequestCommand message, IMessageHandlerContext context)
+        {
+            var cohort = await _dbContext.Value.GetCohortAggregate(message.CohortId, default);
```

Comment:
If easy, can we please check over here if this is a CoP cohort, just for belt & braces.